internal/pkg/template/input/input.go

```diff
@@ -41,14 +41,23 @@ func (i Inputs) ValidateDefinitions() error {
 	return validator.Validate(i, validations...)
 }
 
+const (
+	KindInput       = "input"
+	KindPassword    = "password"
+	KindTextarea    = "textarea"
+	KindConfirm     = "confirm"
+	KindSelect      = "select"
+	KindMultiSelect = "multiselect"
+)
+
 type Input struct {
 	Id          string      `json:"id" validate:"required,template-input-id"`
 	Name        string      `json:"name" validate:"required"`
 	Description string      `json:"description" validate:"required"`
 	Default     interface{} `json:"default,omitempty" validate:"omitempty,template-input-default"`
 	Kind        string      `json:"kind" validate:"required,oneof=input password textarea confirm select multiselect"`
 	Type        string      `json:"type,omitempty" validate:"required_if=Kind input,omitempty,oneof=string int float64,template-input-type"`
-	Options     []Option    `json:"options,omitempty" validate:"required_if=Type select Type multiselect,template-input-options"`
+	Options     []string    `json:"options,omitempty" validate:"required_if=Type select Type multiselect,template-input-options"`
```

Comment:
Je tady ten `Option` typ vůbec potřeba když jde o volně definovatelný string? Oddělal jsem to protože mi někde failoval nějaký test co jsem nedokázal jinak spravit, ale už nevím který. 🙈😃 Ale jestli myslíš že je lepší ho nechat, tak ho dohledám. 🙂

Reply:
Option mala mat `id`, a `name` nie? .... tj, to co sa zobrazi uzivatelovi, a to co sa ulozi (ako klasicky HTML selectbox)

Reply:
Ak sa stratila tato informacia niekde cestou, tak nevadi, spravi sa v dalsom PR.

Reply:
A jo, tak to mi vůbec nedošlo. V jira issue (COM-1248) to teda zmíněný není, ale napadnout mě to mohlo, no. 🙈

Reply:
Tak potom moja chyba, uz som to nestihol checkovat, ci je to v Jire. 
Ale to bude par riadkov.

Pravaze, kedze som tam videl ten typ `Option`, tak som myslel, ze to mas v plane a nerisil som to.
Inak tam ten vlastny typ zmysel fakt nedava.

Reply:
https://keboola.atlassian.net/browse/COM-1266